.github/workflows/ci.yaml

```diff
@@ -300,8 +300,28 @@ jobs:
             -f <(${KBLD} -f bundle/.imgpkg/images.yml -f bundle/config)
         echo "##[endgroup]"
 
-    # TODO add acceptance tests
-    
+    - name: Checkout conformance tests
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 1
+        path: conformance-tests
+        ref: main
```

Comment:
Pin to a tag or a specific sha. Floating dependencies like this can break CI for a future build and it won't be clear why.

Reply:
I agree, I'd rather not have this floating.  I'd like to pin this to a release of the conformance tests, which we currently don't have.  Once we make a release of the conformance test suite, I'll change this.

Reply:
Fixed in 988cc73.